lib/safira_web/live/landing/team_live/components/team.ex

```diff
@@ -0,0 +1,31 @@
+defmodule SafiraWeb.Teamcomponent do
+  @moduledoc """
+  Team component.
+  """
+  use SafiraWeb, :component
+
+  attr :team_name, :string, required: true
+  attr :members, :list, required: true
+
+  def team(assigns) do
+    ~H"""
+    <div class="text-left my-8">
+      <h2 class="text-xl font-bold uppercase text-white mb-2 ml-2"><%= @team_name %></h2>
+      <div class="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3">
+        <div :for={member <- @members} class="flex flex-col items-start">
+          <img
+            src={
+              if member.image,
+                do: Uploaders.Member.url({member.image, member}, :original, signed: true),
+                else: "/images/image_team.png"
+            }
+            alt={"#{member.name}'s photo"}
+            class="w-36 h-36 object-cover"
```

Comment:
```suggestion
            class="w-48 h-48 object-cover"
```

Reply:
@joaodiaslobo @ruioliveira02 can you test this PR with `w-48` in this componenent, me and @AfonsoMartins26 have two completely different perspectives when we change that. In my POV looks great but on his it looks messy, and I don't think it's due to the resolution.